libafl/src/executors/timeout.rs

```diff
@@ -322,11 +296,32 @@ where
     }
 }
 
+#[cfg(unix)]
+impl<E> HasOnCrashReset for TimeoutExecutor<E> {
+    /// Disarm currently running timer
+    fn reset(&self) {
+        unsafe {
+            let disarmed: libc::itimerspec = zeroed();
+            libc::timer_settime(self.timerid, 0, &disarmed as *const _, null_mut());
+        }
+    }
```

Comment:
This should call through to the wrapped `reset` fn in case another Executor wants to also use this API

Reply:
Still, the recursive call to `self.executor.reset_on_crash()` (or whatever we're going to call it)